docs/userGuide/syntax/questions.mbdf

```diff
@@ -1,130 +1,343 @@
-## Questions
+## Questions and Quizzes
 
-**Question component consists of a question body, a hint and an answer.**
+Question and quiz components provide an easy way to test readers on the relevant content topic in the page.
 
-<include src="codeAndOutput.md" boilerplate >
+#### Introduction
+
+Question components (`<question>`) can be one of the following types: **Mcq**, **Checkbox** or **Text**.
+
+In all cases, note that content directly inserted in between `<question>...</question>` will be inserted into the **question body**.
+
+You can also insert markdown into the **header** or **hint box**, by using the `header` and `hint` attributes respectively. Click the hint button below to see how the hint box turns out!
+
+<include src="codeAndOutputSeparate.md" boilerplate >
 <variable name="highlightStyle">html</variable>
+<variable name="heading">Header and hint syntax</variable>
 <variable name="code">
-<question>
-  The question body. <md>:grey_question:</md>
-  <div slot="hint">
-    Question hint. <md>:crystal_ball:</md>
-  </div>
-  <div slot="answer">
-    Question answer. <md>:pencil:</md>
-  </div>
+<!-- Insert markdown into the header and hint using the respective attributes -->
+<question ... header="Which of the following is correct? (**Header**)" hint="Think out of the box! :fas-box:">
```

Comment:
Alternatively, the intro section can simply give an example of each question type without code?

Reply:
It feels a little repetitive as questions are pretty big